scripts/install.sh

```diff
@@ -1,4 +1,3 @@
 #!/usr/bin/env bash
 
-jspm install
-tsd install
\ No newline at end of file
+tsd install
```

Comment:
Zarówno tę komendę, jak i tę wyżej wciągnąłbym bezpośrednio do `package.json`, ale to bardziej moje subiektywne zdanie :)